cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/AppFabricProcessorServiceMain.java

```diff
@@ -98,7 +100,8 @@ protected List<Module> getServiceModules(MasterEnvironment masterEnv,
         new AuditModule(),
         new AuthorizationModule(),
         new AuthorizationEnforcementModule().getMasterModule(),
-        Modules.override(new AppFabricServiceRuntimeModule(cConf).getDistributedModules())
+        Modules.override(new AppFabricServiceRuntimeModule(cConf, EnumSet.of(ServiceType.PROCESSOR))
```

Comment:
Please create an overloaded constructor taking `ServiceType ...` vararg parameter and doing the `EnumSet`. This would make client calls simpler.

Reply:
Done.